README.md

```diff
@@ -28,44 +28,36 @@ If your device is not supported, it will show the following message in the debug
 
 `DEBUG (MainThread) [custom_components.tahoma] Unsupported Tahoma device (internal:TSKAlarmComponent).`
 
-| Somfy uiClass     | Home Assistant platform |
-| ----------------- | ----------------------- |
-| HeatingSystem     | climate                 |
-| Awning            | cover                   |
-| Curtain           | cover                   |
-| ExteriorScreen    | cover                   |
-| Gate              | cover                   |
-| GarageDoor        | cover                   |
-| Pergola           | cover                   |
-| RollerShutter     | cover                   |
-| SwingingShutter   | cover                   |
-| Window            | cover                   |
-| AirSensor         | sensor                  |
-| ElectricitySensor | sensor                  |
-| HumiditySensor    | sensor                  |
-| LightSensor       | sensor                  |
-| TemperatureSensor | sensor                  |
-| DoorLock          | lock                    |
-| OnOff             | switch                  |
-| ContactSensor     | binary_sensor           |
-| OccupancySensor   | binary_sensor           |
-| SmokeSensor       | binary_sensor           |
-| WindowHandle      | binary_sensor           |
-| Light             | light                   |
-
-## Not supported (yet)
```

Comment:
I see you removed this section, do we support those now? or is the list too long so we just don't specify what is not supported?

Reply:
We missed a lot of element in Not Supported, and since we are able to retrieve all the possible elements via the API now, I thought it didn't really make sense to document them here. I added them here as a reference.

Reply:
Ok, let's take this part out.